services/services.go

```diff
@@ -0,0 +1,36 @@
+package services
+
+import (
+	"fmt"
+	"os"
+	"log"
+	"io/ioutil"
+	"net/http"
+	"encoding/json"
+
+	"github.com/javargas/academy-go-q12021/entities"
+	"github.com/javargas/academy-go-q12021/repository"
+)
+
+func GetJobsAPI() (entities.Job, *entities.Error){
+	
+    response, err := http.Get("http://api.dataatwork.org/v1/jobs/26bc4486dfd0f60b3bb0d8d64e001800")
+    if err != nil {
+        fmt.Print(err.Error())
+        os.Exit(1)
```

Comment:
It'd be better to return the error here and exit on the `main` function if possible.